src/wagtail_localize_smartling/utils.py

```diff
@@ -21,19 +21,30 @@ def format_smartling_locale_id(locale_id: str) -> str:
     Format a locale ID for the Smartling API. Wagtail/Django use lower case
     (e.g. "en-us") whereas Smartling uses lower case for the language code and
     upper case for the region, if any (e.g. "en-US").
+
```

Comment:
I missed these somehow 🤦🏼. They are the natural place to add the mappings so +100 for that